imxgesrv1.epfl.ch/codes/dos-6.7.yaml

```diff
@@ -0,0 +1,13 @@
+---
+label: "dos-6.7"
+description: "QuantumESPRESSO 6.7.0 dos.x code"
+input_plugin: "quantumespresso.dos"
+on_computer: true
+remote_abs_path: "/home/software/apps/codes/intel/19.1.3.304/quantum-espresso/6.7.0_MPI_MKL/bin/dos.x"
+computer: "imxgesrv1"
```

Comment:
```suggestion
```

Reply:
Thanks! I also remove the line from `pw.yaml` and `ph.yaml`.